src/shared/lib/kakaoShare.ts

```
@@ -0,0 +1,50 @@
+export const initializeKakao = () => {
+  return new Promise<void>(resolve => {
+    if (window.Kakao && window.Kakao.Link) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)